components/Description-Scott/AddToCart/SelectQuantity.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import DropdownButton from 'react-bootstrap/DropdownButton';
+import Dropdown from 'react-bootstrap/Dropdown';
+
+const SelectQuantity = ({ sku, setQuantitySelected }) => {
+  const [currentQuantity, setCurrentQuantity] = useState(1);
+  const [quantityAvailable, setQuantityAvailable] = useState(0);
+
+  const checkQuantity = () => {
+    if (sku) {
+      setQuantityAvailable(sku.quantity);
+      setCurrentQuantity(1);
+    }
+  };
+
+  const clickHandler = (quantity) => {
+    setQuantitySelected(quantity);
+    setCurrentQuantity(quantity);
+  };
+
+  useEffect(() => {
+    checkQuantity();
+  }, [sku]);
+
+  if (quantityAvailable === 0) {
+    // eslint-disable-next-line vars-on-top
+    var emptyStock = ( // eslint-disable-line no-var
+
+      <DropdownButton disabled id="dropdown-basic-button" title="-" />);
+  } else {
```

Comment:
You want to ? Could start using some ternary operations : keep doing what you're doing.